### PR TITLE
ci/cd: add azure-deploy json params

### PR DESCRIPTION
**Pull request by jdhoffa**
Along with https://github.com/RMI-PACTA/workflow.benchmark.preparation/commit/ae1e91a9b58763f7ece2e04a967b715649d4ae3d which was mistakenly committed directly to main (🤦 ), this PR will allow the benchmark preparation Docker image to be run on Azure. 

Raw benchmarks can be found in this file-share on `pactarawdata`:
https://portal.azure.com/#view/Microsoft_Azure_FileStorage/FileShareMenuBlade/~/overview/storageAccountId/%2Fsubscriptions%2Ffeef729b-4584-44af-a0f9-4827075512f9%2FresourceGroups%2FRMI-SP-PACTA-PROD%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2Fpactarawdata/path/benchmarks/protocol/SMB
Currently it contains only one file, the MSCI raw data. The rest of the benchmarks are scraped as usual. 

Outputs will be saved to this file-share on `pactadatadev`:
https://portal.azure.com/#view/Microsoft_Azure_FileStorage/FileShareMenuBlade/~/overview/storageAccountId/%2Fsubscriptions%2Ffeef729b-4584-44af-a0f9-4827075512f9%2FresourceGroups%2FRMI-SP-PACTA-DEV%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2Fpactadatadev/path/workflow-benchmark-preparation-outputs/protocol/SMB

### CODE REVIEW

**Comment by cjyetman** [comment]
setting to draft until we get necessary repo secrets installed

**Review by AlexAxthelm — APPROVED** [review]

**Review by AlexAxthelm — CHANGES_REQUESTED** [review]
Deploy is failing due to wrong container name

https://github.com/RMI-PACTA/workflow.benchmark.preparation/actions/runs/9302032173/job/25602513038#step:5:39

File: azure-deploy.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            "logType": "ContainerInstanceLogs",
+            "workspaceId": "[parameters('logWorkspaceId')]",
+            "workspaceKey": "[parameters('logWorkspaceKey')]"
+          }
+        },
+        "containers": [
+          {
+            "name": "benchmark-prep-runner",
+            "properties": {
+              "image": "[concat(variables('containerregistry'),'/workflow.scenario.preparation:', parameters('imageTag'))]",
```
Comment (AlexAxthelm) [inline azure-deploy.json:145]:
```suggestion
              "image": "[concat(variables('containerregistry'),'/workflow.benchmark.preparation:', parameters('imageTag'))]",
```

**Review by AlexAxthelm — CHANGES_REQUESTED** [review]

**Review by AlexAxthelm — APPROVED** [review]

**Comment by jdhoffa** [comment]
Ah oops! awesome thank you

**Review by cjyetman — APPROVED** [review]

**Pull request merged**